### ISSUE TITLE
Use restic instead of osm to upload snapshots

### ISSUE THREAD

**Issue by tamalsaha** [kubedb/project#168]
Why?
- Encryption
- Diff for large backups.

**Comment by tamalsaha** [comment]
This work in happening as part of Stash v1beta1 https://github.com/appscode/stash/issues/648

**Comment by tamalsaha** [comment]
[kubedb-stash-integration.pdf](https://github.com/kubedb/project/files/3072612/kubedb-stash-integration.pdf)

**Comment by tamalsaha** [comment]
We are not going to block 0.12.0 release for this. We are making progress in general on this one.

**Comment by codejamninja** [comment]
This looks great. How do I use it?

**Comment by shivpathak** [comment]
@hossainemruz 

#49 was closed on behalf #168, May I know if #49 is done? I wanted to delete snapshots older than 14 days for example.

**Comment by hossainemruz** [comment]
@shivpathak Yes it is done. From next release, using Stash to backup KubeDB databases is recommended.

**Comment by shivpathak** [comment]
@hossainemruz Thanks.

**Comment by shivpathak** [comment]
@hossainemruz what is ETA for next release for kubedb including stash feature?

**Comment by shivpathak** [comment]
> This looks great. How do I use it?

You can refer to documentation https://stash.run/docs/v0.9.0-rc.2/addons/mongodb/guides/3.6/mongodb/
